resources/spec/virkailijaEditorSpec.js

```diff
@@ -22,6 +22,12 @@ function formListItems(n) {
   }
 }
 
+function personInfoModule() {
+  return testFrame()
+      .find('.editor-form__module-wrapper')
+      .find("header:contains('Henkilötiedot')");
```

Comment:
Why is this header needed? If it really is, then one find is enough:
.find(".editor-form__module-wrapper header:contains('Henkilötiedot')")
If the header is what we're after, can we give it a class so that we don't have to use contains?